routes/api.php

```diff
@@ -19,3 +19,8 @@
 });
 
 Route::post('/offers', "OfferController@store");
+
+Route::group(['prefix' => 'v1'], function () {
```

Comment:
@RanaEmad API versioning is a commonly used method when building APIs and a very important approach. I think you need to move the other APIs under version 1 regardless we need adding another version or not.

Reply:
Sure, I would have used it if the project was not ending in a couple of weeks and I do believe it is connected to adding another version. I really don't believe it's necessary, but if you do, you keep your endpoints with versioning and I'll keep mine the way they are, this way we'll both be happy.

Reply:
@RanaEmad  It will not be a good thing to let both of us using a different way. Anyway, I will remove the versioning just for keeping the code consistency although I still prefer using versioning if we see that we can move on with this project to a good step not just for the next couple of weeks.